content/en/api/synthetics/put_test.md

```diff
@@ -15,59 +15,72 @@ A browser test is treated like a GET API test. This method gives you the ability
-
-<table>
- <tr>
-    <th><code>type</code></th>
-    <th>Valid <code>operator</code></th>
-    <th>Value type</th>
-  </tr>
-  <tr>
-    <td>Status Code</td>
-    <td><code>is</code>, <code>is not</code></td>
-    <td>Integer</td>
-  </tr>
-  <tr>
-    <td>Response time</td>
-    <td><code>less than</code></td>
-    <td>Integer</td>
-  </tr>
-  <tr>
-    <td>Headers</td>
-    <td><code>contains</code>, <code>does not contain</code>, <code>is</code>, <code>is not</code>, <code>matches</code>, <code>does not match</code></td>
-    <td>for <code>contains</code>/<code>does not contain</code>/<code>is</code>/<code>is not</code>: String for <code>matches</code>/<code>does not match</code>: <a href="https://docs.datadoghq.com/tagging/using_tags">RegexString</a></td>
-  </tr>
-  <tr>
-    <td>Body</td>
-    <td><code>contains</code>, <code>does not contain</code>, <code>is</code>, <code>is not</code>, <code>matches</code>, <code>does not match</code></td>
-    <td>for <code>contains</code>/<code>does not contain</code>/<code>is</code>/<code>is not</code>: String for <code>matches</code>/<code>does not match</code>: <a href="https://docs.datadoghq.com/tagging/using_tags">RegexString</a></td>
-  </tr>
-</table>
-
-   *   **`target`** - _required API test_ - The expected value for the assertion. For `header`, valid values are any of the valid values for the header key that you define in `property`. For `statusCode`, valid values are valid status codes. For `responseTime`, valid values are the expected response times.
-   *   **`property`** - _optional_ - When you are setting up a `header` `type`, this is required to define the headers parameter key. Valid values are any header keys, like `Content-Type` or `Authorization`.
-*   **`request`** - _required API test_ - An object containing all the necessary information to perform the request to your endpoint.
-   *   **`method`** - _required_ - The method of the API to test. Valid values are `GET`, `POST`, `PUT`, `PATCH`, and `DELETE`. Use `GET` for a browser test.
-   *   **`url`** - _required_ - The endpoint for the API Datadog is testing. For a browser test, the URL for the website Datadog is testing.
-   *   **`timeout`** - _optional_ - When the API request will timeout.
-   *   **`headers`** - _optional_ - Headers in the API request.
-   *   **`body`** - _optional_ The body for the API request. Accepts text strings (including a JSON as a text string). Specify the type using `Content-Type` `property` parameter and the type (like `application/json` or `text/plain` in the `headers` paramater.
-*   **`locations`** - _required_ - A list of the locations that you want the tests to be sent from. At least one value is required, and you can use all locations. For a list of valid locations, use the `GET available locations` method. At least one value is required, and you can use all locations.
-*   **`message`** - _required_ - A description of the test.
+### Arguments
+
+The main arguments for your test are:
```

Comment:
Do similar edits for this file.

Reply:
Done, and pushed to the postman collection file.